src/TLuaInterpreter.cpp

```diff
@@ -1556,17 +1556,27 @@ int TLuaInterpreter::getLineCount(lua_State* L)
 // Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#getColumnNumber
 int TLuaInterpreter::getColumnNumber(lua_State* L)
 {
+    QString windowName = QLatin1String("main");
     Host& host = getHostFromLua(L);
     if (lua_isstring(L, 1)) {
```

Comment:
Oh dear - things are broken - this should be `if (lua_gettop(L)) {` I think now

Reply:
applied.